sonar-db/src/main/java/org/sonar/db/version/v55/CreateInitialSchema.java

```diff
@@ -0,0 +1,918 @@
+        .addColumn(componentUuidCol)
+        .addColumn(statusCol)
+        .addColumn(isLastCol)
+        .addColumn(isLastKeyCol)
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("submitter_login").setLimit(255).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("submitted_at").setIsNullable(false).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("started_at").build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("executed_at").build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("created_at").setIsNullable(false).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("updated_at").setIsNullable(false).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("execution_time_ms").build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("snapshot_id").build())
+        .build());
+    addIndex(context, "ce_activity", "ce_activity_component_uuid", false, componentUuidCol);
+    addIndex(context, "ce_activity", "ce_activity_islast_status", false, isLastCol, statusCol);
+    addIndex(context, "ce_activity", "ce_activity_islastkey", false, isLastKeyCol);
+    addIndex(context, "ce_activity", "ce_activity_uuid", true, uuidCol);
+  }
+
+  private void createCeQueue(Context context) throws SQLException {
+    VarcharColumnDef uuidCol = newVarcharColumnDefBuilder().setColumnName("uuid").setLimit(40).setIsNullable(false).build();
+    VarcharColumnDef componentUuidCol = newVarcharColumnDefBuilder().setColumnName("component_uuid").setLimit(40).build();
+    context.execute(
+      new CreateTableBuilder(getDialect(), "ce_queue")
+        .addPkColumn(newIntegerColumnDefBuilder().setColumnName("id").setIsNullable(false).build(), AUTO_INCREMENT)
+        .addColumn(uuidCol)
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("task_type").setLimit(15).setIsNullable(false).build())
+        .addColumn(componentUuidCol)
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("status").setLimit(15).build())
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("submitter_login").setLimit(255).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("started_at").build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("created_at").setIsNullable(false).build())
+        .addColumn(newBigIntegerColumnDefBuilder().setColumnName("updated_at").setIsNullable(false).build())
+        .build());
+    addIndex(context, "ce_queue", "ce_queue_component_uuid", false, componentUuidCol);
+    addIndex(context, "ce_queue", "ce_queue_uuid", true, uuidCol);
+  }
+
+  private void createFileSources(Context context) throws SQLException {
+    VarcharColumnDef projectUuidCol = newVarcharColumnDefBuilder().setColumnName("project_uuid").setLimit(50).setIsNullable(false).build();
```

Comment:
you could use `VarcharColumnDef.UUID_VARCHAR_SIZE` annd `VarcharColumnDef.UUID_SIZE` to avoid the duplication, but I understand this is extra work with limited valuue